core/src/main/java/org/lflang/graph/InstantiationGraph.java

```diff
@@ -31,10 +31,7 @@
 import java.util.Set;
 import org.eclipse.emf.ecore.resource.Resource;
 import org.lflang.ast.ASTUtils;
-import org.lflang.lf.Instantiation;
-import org.lflang.lf.Model;
-import org.lflang.lf.Reactor;
-import org.lflang.lf.ReactorDecl;
+import org.lflang.lf.*;
```

Comment:
The [style guide](https://google.github.io/styleguide/javaguide.html#s3.3-import-statements) prohibits wildcard imports, so I think this particular change should be reverted.

Reply:
```suggestion
import org.lflang.lf.Instantiation;
import org.lflang.lf.Model;
import org.lflang.lf.Reactor;
import org.lflang.lf.ReactorDecl;
```